jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -251,4 +249,31 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       },
     },
   },
+
+  withClientTLS:: {
+    local api = self,
+
+    config+:: {
+      ClientTLS: {
```

Comment:
I think it's a little awkward to have a separate `tls` and `clientTLS` field (note that the field name should be lowercased rather than `ClientTLS`), since it leads to stuttering: `config.clientTLS.clientCAFile`. It also adds mental overhead to remembering all of the fields names, ie if we have `clientTLS` then why is the other field `tls` rather than `serverTLS`? I think it would be better to nest this key directly in the `tls` field.

Reply:
Done